docs/format_signatures.rst

```diff
@@ -65,35 +69,23 @@ set the :confval:`clang_format_signatures_domain_styles` configuration option.
         "sphinx_immaterial.format_signatures",
     ]
 
-    clang_format_signatures_domain_styles = {
-        "cpp": """{
-        BasedOnStyle: LLVM,
-        ColumnLimit: 68,
-        }""",
-    }
+    object_description_options_patterns = [
+        ("cpp:.*": "clang_format_style": {"BasedOnStyle": "LLVM"}),
+    ]
 
-.. confval:: clang_format_signatures_domain_styles
+.. objconf:: clang_format_style
 
-   Dictionary that specifies the `clang-format style options
-   <https://clang.llvm.org/docs/ClangFormatStyleOptions.html>`__ for each
-   domain.  Formatting is enabled *only* for domains that are listed.
+   Specifies the `clang-format style options
+   <https://clang.llvm.org/docs/ClangFormatStyleOptions.html>`__ as a
+   :python:`dict` (JSON object), or :python:`None` to disable clang-format.
 
    .. literalinclude:: conf.py
       :language: python
       :start-after: # BEGIN: sphinx_immaterial.format_signatures extension options
       :end-before: # END: sphinx_immaterial.format_signatures extension options
```

Comment:
Here is the snippet that has the misleading `append()` call.
```suggestion
   .. code-block:: py

      object_description_options_patterns = [("cpp:.*", {"BasedOnStyle": "LLVM"})]
```

Reply:
Okay, I removed the literalinclude since it is misleading.

Reply:
And actually there was a bug in conf.py (now fixed) that meant clang-format wasn't being used at all.